azurerm/internal/services/containers/data_source_kubernetes_cluster.go

```diff
@@ -176,6 +176,14 @@ func dataSourceArmKubernetesCluster() *schema.Resource {
 							Computed: true,
 						},
 
+						"node_labels": {
+							Type:     schema.TypeMap,
+							Optional: true,
```

Comment:
It doesn't look like this is used to get the resource and as such should be swapped to 
```suggestion
							Computed: true,
```